From e16af9b79c7e2967e6d4b4436595517708fa4918 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Mon, 10 Mar 2014 13:09:56 +0000
Subject: [PATCH 23/28] thd_model.cpp: remove ununsed variable 'file'

variable 'file' is not used in store_set_point() so remove it

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 src/thd_model.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/thd_model.cpp b/src/thd_model.cpp
index 0b01512..8820386 100644
--- a/src/thd_model.cpp
+++ b/src/thd_model.cpp
@@ -187,7 +187,6 @@ void cthd_model::add_sample(int temperature) {
 
 void cthd_model::store_set_point() {
 	std::stringstream filename;
-	std::ofstream file;
 
 	filename << TDRUNDIR << "/" << "thermal_set_point." << zone_type << "."
 			<< "conf";
-- 
1.9.0

